library/src/components/layout/Header/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -85,34 +93,50 @@ const Breadcrumbs = (props: BreadcrumbsProps): JSX.Element | null => {
     );
   }
 
+  const breadcrumbs: JSX.Element[] =  [];
+  
+  routeItems.forEach((routeItem, key: number) => {
+    const to = getEntityItemLink(routeItem, match);
+    const entity = routeItem.entity;
+    const isLast = key + 1 === routeItems.length;
+    const params = Object.values(match.params);
+    const entityId = params[key];
+
+    const element = (
+      <span key={key} style={{ display: 'flex', gap: '12px' }}>
+        <StyledCollapsedBreadcrumbsLink to={to}> 
+          {entity.title}
+        </StyledCollapsedBreadcrumbsLink>
+        {isLast && entity.link && !appendSegment && (
+          <a
+            target='_blank'
+            href={entity.link}
+            rel='noreferrer'
+            style={{ height: '24px' }}
+          >
+            <img src='assets/img/breadcrumb-link.svg' />
+          </a>
+        )}
+      </span>
+    );
+
+    breadcrumbs.push(element);
+
+    if (isLast || !entityId) {
+      return;
+    }
+    
+    breadcrumbs.push(
+      <BreadcrumbItem key={key} entity={entity} id={entityId as string} />
+    );
```

Comment:
Instead of manually pushing elements in different cases, return an array from .map() using Fragment where needed.

```suggestion
    return (
    <React.Fragment key={key}>
     <span style={{ display: 'flex', gap: '12px' }}>
     .
     .
     .
     </span >
     
      {!isLast && entityId && (
        <BreadcrumbItem entity={entity} id={entityId as string} />
      )}
     </React.Fragment>
    )

```

Reply:
But I need them two be separate array items. That's the whole point of this refactor. If I have this ["a", "b", "c"], MuiBreadcrum will generate:
` <li>a</li>
 <li>b</li>
 <li>c</li>
`

But if I do ["a b", "c"] I'll get:
`
<li>a b</li>
<li>c</li>
`